crates/orchestrator/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "orchestrator"
```

Comment:
Can we give this a proper prefix, e.g. mysten-aws-orchestrato, sui-aws-orchestrator or aws-orchestrator? For example, there are already components named TransactionOrchestrator.

Reply:
Yeah it would be nice to have a little more descriptive name